### ISSUE TITLE
FOLA: 2025 Leads/PM Agenda

### ISSUE THREAD

**Issue by MuthamilselvanG** [hackforla/food-oasis#2227]
- This is closed for 2025.  
- For 2026 agendas, go to #2613 

### Overview

This issue tracks the agenda for our weekly meetings

**Issue Template**

```
## [ Date ]  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       - [ ] [Tiffany](https://github.com/hackforla/food-oasis/issues/assigned/TifNi)
       - [ ] [Payel](https://github.com/hackforla/food-oasis/issues/assigned/pbandyop)
       - [ ] [Alexa](https://github.com/hackforla/food-oasis/issues/assigned/Ray65)
       - [ ] [Saumil](https://github.com/hackforla/food-oasis/issues/assigned/SAUMILDHANKAR)
       - [ ] [Joshua](https://github.com/hackforla/food-oasis/issues/assigned/joshsanchez98)
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items
```

### Links to other agenda issues for your team

(so that you can easily post on their agendas items that come up at yours)

- Agenda: [FOLA: Design](https://docs.google.com/document/u/1/d/1oLPGqa2DX0tvGLc2F3mN1xqmMjZurUYd4-AKkMj4bWI/edit?usp=sharing)
- Agenda: [FOLA: PM Agenda (Google Doc - Deprecated)](https://www.google.com/url?q=https://docs.google.com/document/d/1uDexsQ1BQNzk0iohSMa_tglt9TRwSwh8IsJJsnW7c-c/edit%23heading%3Dh.ptpx69qsotgg&sa=D&source=calendar&ust=1726938230027467&usg=AOvVaw2ho9q05jZALwYRnYLuCQtR) and [FOLA: PM Agenda (GitHub Issue - Current)](https://github.com/hackforla/food-oasis/issues/2226)
- Agenda: [FOLA: Leads/PM (Google Doc - Deprecated)](https://docs.google.com/document/u/1/d/1uPO0_AJ1zn3IZXrhHBXub-ZfakIsx34jKHK3LIB8VmM/edit)
- Agenda: [FOLA: UXR](https://docs.google.com/document/d/1ETqJsxCE_HnR5PluaCiTry_iDkv0Uy7f0wo6xHujWvU/edit)

**Comment by farahrobleh** [comment]
## 2024-09-26 Meeting Agenda

### Prework to prep for meeting
- [ ] Ask John where open PRs are supposed to live on the project board. (Reference: https://github.com/orgs/hackforla/projects/59/views/1?filterQuery=is%3Apr)
- [ ] Do they still need this column?: https://github.com/orgs/hackforla/projects/59/views/1?filterQuery=status%3A%22In+Development%22
- [ ] add a link to the most recent label check to the agenda

### Recurring items: Happens on the Wednesday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/3?filterQuery=label%3A%22Product+Management%22+-status%3AReference%2C%22In+Development%22++status%3A%22New+Issue+Approval%22)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Robleh](https://github.com/hackforla/food-oasis/issues/assigned/farahrobleh)
       - [ ] [Sumit](https://github.com/hackforla/food-oasis/issues/assigned/sumit-sharma92)
       - [ ] [Muthamil](https://github.com/hackforla/food-oasis/issues/assigned/MuthamilselvanG)
       - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by ExperimentsInHonesty** [comment]
Notes from 2024-09-26  meeting
- removed the In Development column
- removed the ready for dev lead label from all closed issues
- prioritized all the issues in the new issue approval column into the ER / Epic ready to be made into issues column

**Comment by ExperimentsInHonesty** [comment]
## 2024-10-03 Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Robleh](https://github.com/hackforla/food-oasis/issues/assigned/farahrobleh)
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [Sumit](https://github.com/hackforla/food-oasis/issues/assigned/sumit-sharma92)
       - [ ] [Muthamil](https://github.com/hackforla/food-oasis/issues/assigned/MuthamilselvanG)
       - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by farahrobleh** [comment]
## 2024-10-10 Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Robleh](https://github.com/hackforla/food-oasis/issues/assigned/farahrobleh)
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [Sumit](https://github.com/hackforla/food-oasis/issues/assigned/sumit-sharma92)
       - [ ] [Muthamil](https://github.com/hackforla/food-oasis/issues/assigned/MuthamilselvanG)
       - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by sumit-sharma92** [comment]
2024-10-17 Meeting Agenda

Prework to prep for meeting


Recurring items: Happens on the Thursday meeting

 review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)

 Accountability and Support Check.

 Review assignments for each Team member
 [Robleh](https://github.com/hackforla/food-oasis/issues/assigned/farahrobleh)
 [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
 [Sumit](https://github.com/hackforla/food-oasis/issues/assigned/sumit-sharma92)
 [Muthamil](https://github.com/hackforla/food-oasis/issues/assigned/MuthamilselvanG)
 [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
 [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech)
 [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
 [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
 [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
 [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
 [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
 finish adding labels to the issue that are still missing them (see audit links above)

 review issues with [epic label] and define how to break down further (if necessary)

 Review to see if there are any members to Onboard or Offboard
- [ ] #

New Items

[ ]
 Carry over Items to next meeting that don't get addressed
FYIs

Notes from Meeting

Task Items

**Comment by farahrobleh** [comment]
## 2024-10-24 Meeting Agenda

### Prework to prep for meeting
- [Current Location Button Fails to Update Location Bar](https://github.com/hackforla/food-oasis/issues/2261) issue has been created.
- Review the FOLA team roster, validate which members are still active versus inactive.
- Review https://github.com/hackforla/food-oasis/issues/1409 - get more context on issue/determine if it's really a bug fix.
- From the website team, we have a new issue saying that a new wins form submission has been created, and needs to be reviewed. Review issue https://github.com/hackforla/website/issues/5975
- Review this issue (which has currently been moved to New Issue Approval): https://github.com/hackforla/food-oasis/issues/1128 -  seeking clarity on why we need to sync with existing partners?
- Review this issue: https://github.com/hackforla/food-oasis/issues/1131 - what is the current state of the Suggestions page/Suggestions Administration board and modals, and would this be considered expanding on an existing feature, or creating a new one?

### Recurring items: Happens on the Thursday meeting

- [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)

 - [ ] Accountability and Support Check.

 - [ ] Review assignments for each Team member
    - [ ] [Robleh](https://github.com/hackforla/food-oasis/issues/assigned/farahrobleh)
    - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
    - [ ] [Sumit](https://github.com/hackforla/food-oasis/issues/assigned/sumit-sharma92)
    - [ ] [Muthamil](https://github.com/hackforla/food-oasis/issues/assigned/MuthamilselvanG)
    - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
    - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech)
    - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
    - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
    - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
    - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
    - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)

- [ ] finish adding labels to the issue that are still missing them (see audit links above)

- [ ] review issues with [epic label] and define how to break down further (if necessary)

- [ ] Review to see if there are any members to Onboard or Offboard
   - [ ] #

### New Items
[ ]
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs

### Notes from Meeting




### Task Items

**Comment by sumit-sharma92** [comment]
## 2024-11-07  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Robleh](https://github.com/hackforla/food-oasis/issues/assigned/farahrobleh)
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [Sumit](https://github.com/hackforla/food-oasis/issues/assigned/sumit-sharma92)
       - [ ] [Muthamil](https://github.com/hackforla/food-oasis/issues/assigned/MuthamilselvanG)
       - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] Ask John about creating column in database (if doesn't exist already) for out of state listings like from Texas, Hawaii or NC - so we can tag the listings from those places.
- [ ] Ask John if database has data for US congress and Senate and state assembly and senate persons and city council persons?
- [ ] https://docs.google.com/spreadsheets/d/1gSkcGyxeN-p06uIWceKpGrCs5YNs8x4tYoP_rA8WkoU/edit?gid=0#gid=0&fvid=1071295059 - The sheet shows about 1300 rows but the website has about 600 listings only
- [ ] Need a volunteer to go through the addresses in the csv file and find out which NC it belongs to 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by sumit-sharma92** [comment]
## [ 2024-11-14 ]  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Robleh](https://github.com/hackforla/food-oasis/issues/assigned/farahrobleh)
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [Sumit](https://github.com/hackforla/food-oasis/issues/assigned/sumit-sharma92)
       - [ ] [Muthamil](https://github.com/hackforla/food-oasis/issues/assigned/MuthamilselvanG)
       - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] FOLA:Application User permissions - Process for exporting the user permissions data to a spreadsheet.
- [ ] Diff b/w admin and coordinator roles for volunteers on FOLA:Application
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by fancyham** [comment]
We need to talk about the $5,000 KARST grant, which will expire in a few weeks if we don’t claim it.  If we are not going to take it, we need to let them know.

**Comment by sumit-sharma92** [comment]
## 2024-11-21  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Robleh](https://github.com/hackforla/food-oasis/issues/assigned/farahrobleh)
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [Sumit](https://github.com/hackforla/food-oasis/issues/assigned/sumit-sharma92)
       - [ ] [Muthamil](https://github.com/hackforla/food-oasis/issues/assigned/MuthamilselvanG)
       - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] Bonnie -> We need to start adding appropriate labels to the new issues while creating them. 
          - https://github.com/hackforla/food-oasis/issues?q=is%3Aopen+is%3Aissue+label%3A%22Missing%3A+Role%22
          -   https://github.com/hackforla/food-oasis/issues?q=is%3Aopen+is%3Aissue+no%3Amilestone+label%3A%22Missing%3A+Milestone%22
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by jelenaUX** [comment]
We have received a survey response from a person who is helping Food Cycle explore integration with FOLA website. This person left an email for follow-up: details in a [Slack link](https://hackforla.slack.com/archives/CMER3R1RD/p1733433758306869). 

Is anyone on the development team, project managers, leads, already in touch with Food Cycle? Should I try to get in touch with this person? I'd like to hear if they are experiencing any issues with the FOLA widget, and whether they want our pop-up survey to appear in FOLA widget.

**Comment by fancyham** [comment]
- Talked about the hot-fix that manually added announcement to all pages re: ‘fire and some info may be outdated’.  Doesn’t look great on mobile, but works for now.
- Talked about long-term replacement for this and worked through designs and functionality.
  - Bryan: Created issue: https://github.com/hackforla/food-oasis/issues/2326 (edited)

**Comment by fancyham** [comment]
## [ Date ]  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Robleh](https://github.com/hackforla/food-oasis/issues/assigned/farahrobleh)
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [Sumit](https://github.com/hackforla/food-oasis/issues/assigned/sumit-sharma92)
       - [ ] [Muthamil](https://github.com/hackforla/food-oasis/issues/assigned/MuthamilselvanG)
       - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] Updates on connecting with LA foodcycle — last email was Jan 13 from Nancy Beyda asking for a meeting.   Seems like FoodCycle wants to work with us to add their missing pantries (perhaps what they mean by 'integration'). (John leading this effort?
- [ ] Updates on UCLA volunteers cohort — who is managing it?
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by Rabia2219** [comment]
#### LettuceMeet Link

- https://lettucemeet.com/l/0MNAJ

**Comment by GigiUxR** [comment]
Please add this to today's agenda

/////////////////////////////////////
TIME SENSITIVE
//////////////////////////////////

Subject: Prioritizing Data Validation for “No Known Eligibility Requirements” Listings
    
Hi  @sumit-sharma92     @ExperimentsInHonesty 
     
The following was discussed at our research meeting today. Given the current immigration enforcement climate, we anticipate an increased need among undocumented individuals seeking food assistance without documentation requirements.
    
**Identified Issue**
Currently, the “No Known Eligibility Requirements” filter could serve as a proxy for this user group. However, data accuracy and validation gaps make it unclear whether all linked listings truly meet this criterion. Incomplete or outdated information risks creating friction for users in crisis, potentially leading to failed searches and diminished trust in our platform.
      
**Recommendation**
**We propose prioritizing these listings for validation.**

**Comment by GigiUxR** [comment]
@sumit-sharma92  and @ExperimentsInHonesty 

On a related note.... @ExperimentsInHonesty @sumit-sharma92 

Earlier today , I analyzed our eligibility data and identified a data consistency issue in our directory related to eligibility information. Currently, eligibility details are intermingled with intake requirements and service notes, leading to inconsistencies in how information is structured and presented.
                                                
**Key Issues Identified**
Blurring of Critical Categories:
Eligibility (who qualifies) is frequently mixed with intake requirements (what must be provided upon arrival) and service notes (miscellaneous information, e.g., appointment needed).
This inconsistency makes it difficult to ensure we fully capture all necessary details for each category, which could lead to incomplete or misleading information for users.
Broad vs. Granular Information:
Some listings use general terms like “proof of income,” while others specify exact documents like a W-2 or pay stub.
Without standardization, it’s unclear whether “proof of income” means any form of documentation is accepted or if specific documents are required.
This inconsistency can result in missing key details, making it harder for users to determine whether they qualify for services.
                                                     
**Proposed Solution:** Volunteer Training & Decision Support Tools
To improve data accuracy and usability, I propose developing:
A Standardized Checklist for Volunteers 
A few new slides to educate volunteers on the distinct broad categories and the most frequent eligibility and intake subcategories based on current data trends. 
It will ensure each category is captured distinctly and guide volunteers on where more specificity is required (e.g., listing exact types of accepted documents under proof of income).
A Decision Tree to Guide Validation Calls
A structured decision tree will help.   volunteers systematically ask the right questions when validating pantry listings.
This will ensure we capture missing details, prevent assumptions, and standardize how we collect eligibility and intake data.
Automated Timestamping of Data Entries
Implementing field-level timestamps would allow us to track when each piece of information was last validated.
This would improve data quality assurance and help prioritize outdated listings for review.
         
I can execute on the first two items (standardized checklist and decision tree) and have it ready  before Wednesday. The third item will take a bit longer to achieve.

**Comment by fancyham** [comment]
## [ Date ]  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Robleh](https://github.com/hackforla/food-oasis/issues/assigned/farahrobleh)
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [Sumit](https://github.com/hackforla/food-oasis/issues/assigned/sumit-sharma92)
       - [ ] [Muthamil](https://github.com/hackforla/food-oasis/issues/assigned/MuthamilselvanG)
       - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       

- [ ] Eligibility validation and checklist that Gigi describes above
- [ ] Marketing approvals — set up and kick off marketing efforts
- [ ] Leads meeting reschedule?
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by ExperimentsInHonesty** [comment]
notes john will call nancy or bonnie will email next week
gigi needs to redo her hours in the lettucemeet because right now it shows 7:30 pm only on two days

**Comment by sumit-sharma92** [comment]
Wednesday 2pm - time decided for All team meeting starting 25th February

**Comment by fancyham** [comment]
2/20/2025 Meeting Agenda

Prework to prep for meeting

 #
Recurring items: Happens on the Thursday meeting

 review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)

 Accountability and Support Check.

 Review assignments for each Team member
 [Robleh](https://github.com/hackforla/food-oasis/issues/assigned/farahrobleh)
 [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
 [Sumit](https://github.com/hackforla/food-oasis/issues/assigned/sumit-sharma92)
 [Muthamil](https://github.com/hackforla/food-oasis/issues/assigned/MuthamilselvanG)
 [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
 [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech)
 [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
 [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
 [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
 [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
 [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
 Eligibility validation and checklist that Gigi describes above

 Marketing approvals — set up strategy and kick off marketing efforts

 Leads meeting reschedule?

 Who is usually responsible for managing this meeting’s agenda?

 finish adding labels to the issue that are still missing them (see audit links above)

 review issues with [epic label] and define how to break down further (if necessary)

 Review to see if there are any members to Onboard or Offboard
- [ ] #

New Items

[ ]
 Carry over Items to next meeting that don't get addressed
FYIs

Notes from Meeting

Task Items

**Comment by fancyham** [comment]
> Wednesday 2pm - time decided for All team meeting starting 25th February

The 25th is a Tuesday, which is what John and I recall we changed it to, so I’ll update the calendar

**Comment by fancyham** [comment]
## 2/20/2025  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Robleh](https://github.com/hackforla/food-oasis/issues/assigned/farahrobleh)
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [Sumit](https://github.com/hackforla/food-oasis/issues/assigned/sumit-sharma92)
       - [ ] [Muthamil](https://github.com/hackforla/food-oasis/issues/assigned/MuthamilselvanG)
       - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       

- [ ] Eligibility validation and checklist that Gigi describes above
- [ ] Marketing approvals — set up strategy and kick off marketing efforts
- [ ] Leads meeting reschedule?
- [ ] Who is usually responsible for managing this meeting’s agenda?
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting
- New feature proposal - https://github.com/hackforla/food-oasis/issues/2117

### Task Items
- Decide the meeting time starting next week to match other team members' availability
- Focus on Altadena listings validation

**Comment by GigiUxR** [comment]
**### 03/05/2025 Meeting Agenda**


**Notes from Meeting**

- PM & Issues board
      John mentioned that the issue board is getting a bit hairy! Bonnie explained that 
      Validation Volunteer management is using up all of our current  PM's time.   She has 
      solutions for picking back up the momentum by introducing new type of 
      PMs with 20-hour week availability (but likely short-term availability) and a 
      designated student volunteer manager who can dedicate themselves with us as a 
      stakeholder longterm.

- We want to grow our database with new listings. 
     
    - John D is continuing his next steps with LA FoodCycle.  
    - GigiUxR mentioned Mutual Aid LA' s site for listings. John said that they might 
       be worth exploring as an opportunity for new partner.  Bonnie suggested a data 
       science contact (see below in tasks) at Slack that could help us attain listings.
        
- Issue #2346
    - GigiUxR briefly described first steps to avoid overlap with Bryan's review of eligibility 
       data. (The process is detailed in Slack chat earlier this day.)
    - John D suggested as a possible resource an organization focused on standards / 
       taxonomy: https://openreferral.org/  

**Task Items**

- Reconfirm next steps regarding Mutual Aid LA, is GigiUxr to contact Sophia Alice, Data 
   Science Slack Channel for scraping code?
- Bonnie to bring in new 20-hr a week PMs
- Bonnie to post an ad for student volunteer program manager
- GigiUxR to access https://openreferral.org/  
- GigiUxR to bring UxR dashboard sheet next meeting so Bonnie can help integrate the 
   dashboard into a github alternative.

**Comment by ExperimentsInHonesty** [comment]
## 2025-03-11  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Robleh](https://github.com/hackforla/food-oasis/issues/assigned/farahrobleh)
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [Sumit](https://github.com/hackforla/food-oasis/issues/assigned/sumit-sharma92)
       - [ ] [Muthamil](https://github.com/hackforla/food-oasis/issues/assigned/MuthamilselvanG)
       - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by ExperimentsInHonesty** [comment]
Notes: 

- Summit availability
   - Student status
- Bonnie to begin process for bringing in new 20-hr a week PMs
   - one more week on the process and I will know if we can get pms that way

**Comment by neom500** [comment]
Notes:

Nate (Won't be able to attend today's meeting but here are a list of open issues I wasn't sure if they fall as a Food Seeker issue, Admin Dashboard issue, or both):
- 2344 (backend but linked to both food seeker/admin views 
- 2339 (same as 2344)
- 2313
- 2252
- 2251
- 2172

**Comment by fancyham** [comment]
Would like to discuss: 
- want to add text tags to Eligibility Requirements field like “serves-veterans”, “serves-under-18” — to call out populations a location explicitly says they serve.  Similar text-tags for requirements; also, 
- split Eligibility Requirements field into two separate fields: Eligibility, and Requirements

**Comment by nataliecly** [comment]
### For April 22 meeting :

Onboarding is happening on 2025-04-28 and currently [here are the open roles](https://github.com/orgs/hackforla/projects/67/views/1?filterQuery=fola). If you want more people, please make new open role issues

**Comment by fancyham** [comment]
Bryan : after the meeting, I created #2421 to add all UI element styles to the sampler page

**Comment by nataliecly** [comment]
## 04/29/2025  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       - [ ] [Natalie](https://github.com/hackforla/food-oasis/issues/assigned/nataliecly)
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by ExperimentsInHonesty** [comment]
- turned on summary
- BA's will work on better categorization of the existing food listings on NCs
- BA's looking for food insecurity data that is more relevant
- Dev, is working on bug fixes, typescript and other non-urgent issues
   - Dev, needs new issues to work on 
- Student validation: the instructions are improving 
- Student validation: we will need to improve the interface to show error messages that show what still has to happen before something can be confirmed
- Research: coding the free-form eligibility data we have collected
- Product: Consent guides will be done by May and the 
- Design: To be able to search on organization names

**Comment by fancyham** [comment]
My (not AI) meeting notes:

- New product managers  
  - Tiffany Sia  
  - Sophie Frolik (student intern for one quarter(?)) \- helping revise validation instructions, improve UI  
  - Stella Mao  
  - Chanel Johnson  
  - Natalie Yu (still working on Cal State Long Beach(?) student transit layer project)  
- Shienny  
  - Gave update on dashboard progress  
- John D  
  - Update on dev team  
    - Updating client to Typescript  
- Sophie (validation usability and instruction updates)  
  - Update: Found a usability issue with validation – ‘approve’ button looked disabled but didn’t know why – it’s because all ‘confirm’ checkboxes must be checked… will write an issue, document other problems they find  
- Natalie (transit layer data processing)  
  - Update: Student not making good progress  
- Gigi  
  - Asked about if can start survey soon : Bonnie asks to wait till HfLA consent boilerplate is done at end of May to review ourselves to see if anything needs to be updated  
  - Will share in-progress thoughts next week about eligibility/requirements ontology/tagging  
- Bryan  
  - We’re working on more friendly, quality of life stuff, doing housekeeping to make it easier to build in the future  
  - Org Name search is still in progress, but hopefully soon.  Checking in with Jelena  
  - Will be on vacation next week

**Comment by TifNi** [comment]
## 2025-05-06 Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       - [ ] [Natalie](https://github.com/hackforla/food-oasis/issues/assigned/nataliecly)
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by SophieF12** [comment]
I won't be able to attend the meeting today but below are my updates:
- Validation: Out of ~300 listings that need to be approved or disapproved, I estimate that ~90 will be able to be approved without further research/validation. I am about halfway done with these and will have approved all acceptable validations by next Tuesday 5/13. 
- Interface improvements: I posted a demonstration of the user experience when checkboxes are left unchecked in validation along with a proposed error message to the corresponding issue.

**Comment by ExperimentsInHonesty** [comment]
- Bryan gave us an update on #1551

**Comment by ExperimentsInHonesty** [comment]
three new features
- timestamp (existing feature issue)
   - to be used by validation manager, At a glance be able to see if specific fields have been updated data, to know if we want to re verify
- entire record change side by side view (see this issue for the api https://github.com/hackforla/food-oasis/issues/535), create a new design issue
  - to be used by validation manager to approve listings
- field by field change review Could be linked to timestamp (new issue for design)
   - to be used by anyone on the team who wants to see past changes to a specific field.  Validation manager would use this if we were concerned with a specific goal for a quarter, and were doing quick evaluation of submitted listings (e.g., if we asked validators to update all the temporary closed listings).

**Comment by ExperimentsInHonesty** [comment]
![Image](https://github.com/user-attachments/assets/b0343dae-d3e9-438c-a18c-ce175518a41f)

**Comment by TifNi** [comment]
## 2025-05-20  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       - [ ] [Natalie](https://github.com/hackforla/food-oasis/issues/assigned/nataliecly)
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] Items from 5/13/25 Meeting Agenda
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by wmao8** [comment]
## 2025-05-27  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       - [ ] [Natalie](https://github.com/hackforla/food-oasis/issues/assigned/nataliecly)
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] Items from 5/20/25 Meeting Agenda
- [ ] Carry over Items to next meeting that don't get addressed
- [ ] This issue https://github.com/hackforla/food-oasis/issues/2331
> - in particular (this comment: "_HOLD and add to 2pm tuesday team agenda - https://github.com/hackforla/food-oasis/issues/1427_")

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by ExperimentsInHonesty** [comment]
Bryan's updates: Designers are working on Design System and a style guide
Shienny's updates: Need the data

**Comment by ExperimentsInHonesty** [comment]
GIGI, please do not use powerpoint.  Please only use Google Drive, and Create the file or move the file into Google Drive, before adding content to it.  That way the detail is owned by the project, and there is no need to fix formatting after converting.

**Comment by ChanelYJ** [comment]
## 2025-6-10  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       - [ ] [Natalie](https://github.com/hackforla/food-oasis/issues/assigned/nataliecly)
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] https://github.com/hackforla/food-oasis/issues/2211#issuecomment-2840337265
- [ ] [https://github.com/hackforla/food-oasis/issues/2211#issuecomment-294221694](https://github.com/hackforla/food-oasis/issues/2211#issuecomment-2942216947)

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by wmao8** [comment]
## 2025-6-17  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       - [ ] [Natalie](https://github.com/hackforla/food-oasis/issues/assigned/nataliecly)
       - [ ] [Mark] (TBU_cannot the github handle in git)
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [x]  #

### New Items


### FYIs
-

### Notes from Meeting



### Task Items

**Comment by wmao8** [comment]
## 2025-6-24  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       - [ ] [Natalie](https://github.com/hackforla/food-oasis/issues/assigned/nataliecly)
       - [ ] [Mark] [TBU_cannot find the github handle on git or roster https://docs.google.com/spreadsheets/d/1oM4ZRq0d2ukPQusZ7p7wDqgIuuaENyUgNT3QrUFfDXg/edit?gid=2032152864#gid=2032152864)]
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [x]  #

### New Items




### FYIs
-

### Notes from Meeting



### Task Items

**Comment by pbandyop** [comment]
## 2025-8-11  Meeting Agenda

### Prework to prep for meeting
- [ ] Discuss this issue: https://github.com/hackforla/food-oasis/issues/2449 

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       - [ ] [Natalie](https://github.com/hackforla/food-oasis/issues/assigned/nataliecly)
       - [ ] [Mark] - not yet on roster
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items

- [x] Have Mark fill out 
   - [ ] [ROSTER Food Oasis Team](https://docs.google.com/spreadsheets/d/1--JQ5WTLpQcBkSC5wbvQKFZ3Mw8OoHK9ToyXrymVGnM/edit?gid=0#gid=0) 
   - [ ] [Product Management CoP Roster Assigned and Former](https://docs.google.com/spreadsheets/d/1oM4ZRq0d2ukPQusZ7p7wDqgIuuaENyUgNT3QrUFfDXg/edit?gid=2032152864#gid=2032152864) 
- [x] https://github.com/orgs/hackforla/projects/59/views/2?filterQuery=-status%3AReference%2C%22In+Development%22+label%3A%22dev%22%2C%22Role%3A+Back-end%22%2C%22Role%3A+DevOps%22%2C%22Role%3A+Front-end%22+-label%3A%22Interface%3A+Food+Seeker%22+label%3A%22Ready+for+dev+lead%22 - Dev lead please review this issues as a top priority, so that the PMs and I who are working on the Admin Dashboard issues, can prioritize all the open issues and start making new ones
- [ ] Discuss https://github.com/hackforla/food-oasis/issues/1986
   - Cons would be:
   - Allowing Food Seeker users to be authenticated so they can represent a pantry and edit their own listing(s), publish news about their listings, like special events, etc.
   - Allowing users to sign up for custom notifications of some sort (changes to pantries, special events, etc.)
   - Allowing users to save their preferences (favorite pantries, search locations, search preferences, ratings, etc.)
- [ ] https://github.com/hackforla/food-oasis/issues/1273#issuecomment-2964669651
- [ ] Discuss: https://github.com/hackforla/food-oasis/issues/2435#issue-304745895



### FYIs
-

### Notes from Meeting



### Task Items

**Comment by TifNi** [comment]
## 2025-08-19  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       - [ ] [Natalie](https://github.com/hackforla/food-oasis/issues/assigned/nataliecly)
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by ExperimentsInHonesty** [comment]
bryan will be gone from 2025-09-11 to 10-23
john will be gone from 2025-09-17 09-27

cal state long beach via slack

**Comment by Ray65** [comment]
## 2025-11-4 Meeting

@TifNi and @ExperimentsInHonesty - Request to please review my stories (links below) and share your feedback and next steps with me. Thank you!

**Corrections**
1. Add fields: "Date Created", "Number of Days Open"
https://github.com/hackforla/food-oasis/issues/2580
Question: Might be useful to add "Date Closed" to Suggestions Admin as well?

2. Default Sort Order: Newest First on Corrections & Suggestions Admin Page
https://github.com/hackforla/food-oasis/issues/2575

3. Simplify and Standardize “Status” Field
https://github.com/hackforla/food-oasis/issues/2581

4. Allow ability to sort all columns in Suggestions Admin
https://github.com/hackforla/food-oasis/issues/2585

5. Expand Filtering Capabilities in Suggestions Admin
https://github.com/hackforla/food-oasis/issues/2586
Note: If "Date Closed" is added to Suggestions Admin (per 1b), filtering capability must also be added to it ==> update this story.

6. Update heading of Suggestions Admin
https://github.com/hackforla/food-oasis/issues/2587


**Verification Admin Criteria Window**
7. Add capability to filter based on "Status" in the Criteria Window
https://github.com/hackforla/food-oasis/issues/2574

**Comment by TifNi** [comment]
## 2025-11-04  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [Nate](https://github.com/hackforla/food-oasis/issues/assigned/neom500)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Qiqi Zheng](https://github.com/hackforla/food-oasis/issues/assigned/qiqicodes)
       - [ ] [Hannah](https://github.com/hackforla/food-oasis/issues/assigned/hanapotski)
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       - [ ] [Natalie](https://github.com/hackforla/food-oasis/issues/assigned/nataliecly)
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] Have John look through ready for dev lead label: https://github.com/hackforla/food-oasis/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22Ready%20for%20dev%20lead%22
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by TifNi** [comment]
## 2025-11-10  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       - [ ] [Tiffany](https://github.com/hackforla/food-oasis/issues/assigned/TifNi)
       - [ ] [Payel](https://github.com/hackforla/food-oasis/issues/assigned/pbandyop)
       - [ ] [Alexa](https://github.com/hackforla/food-oasis/issues/assigned/Ray65)
       - [ ] [Saumil](https://github.com/hackforla/food-oasis/issues/assigned/SAUMILDHANKAR)
       - [ ] [Joshua](https://github.com/hackforla/food-oasis/issues/assigned/joshsanchez98)

- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by TifNi** [comment]
## 2026-01-06  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the Thursday meeting
 - [ ] review any issues that are in the [new issue approval column](https://github.com/orgs/hackforla/projects/59/views/1)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Bonnie](https://github.com/hackforla/food-oasis/issues/assigned/experimentsinhonesty)
       - [ ] [John Darragh](https://github.com/hackforla/food-oasis/issues/assigned/entrotech) 
       - [ ] [Bryan Wu](https://github.com/hackforla/food-oasis/issues/assigned/fancyham)
       - [ ] [Gigi P](https://github.com/hackforla/food-oasis/issues/assigned/GigiUxR)
       - [ ] [Shienny](https://github.com/hackforla/food-oasis/issues/assigned/Shienny1)
       - [ ] [Tiffany](https://github.com/hackforla/food-oasis/issues/assigned/TifNi)
       - [ ] [Payel](https://github.com/hackforla/food-oasis/issues/assigned/pbandyop)
       - [ ] [Alexa](https://github.com/hackforla/food-oasis/issues/assigned/Ray65)
       - [ ] [Saumil](https://github.com/hackforla/food-oasis/issues/assigned/SAUMILDHANKAR)
       - [ ] [Joshua](https://github.com/hackforla/food-oasis/issues/assigned/joshsanchez98)
       
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further (if necessary)
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] Go through the 1Password drive in #2600 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by Rabia2219** [comment]
- https://github.com/hackforla/food-oasis/issues/2613

**Comment by fancyham** [comment]
- This agendas list is closed for 2025.  Continues for 2026 in  #2613
